src/controllers/ProjectController.ts

```diff
@@ -0,0 +1,115 @@
+import { Request, Response } from "express";
+import { getRepository } from "typeorm";
+import { validate } from "class-validator";
+
+import { Project } from "../entities/Project";
+
+class ProjectController {
+  static getAll = async (req: Request, res: Response) => {
+    const projectEntityRepo = getRepository(Project);
+    const users = await projectEntityRepo.find();
+
+    res.send(users);
+  };
+
+  static getOneById = async (req: Request, res: Response) => {
+    const id: number = req.params.id;
+
+    const projectEntityRepo = getRepository(Project);
+    try {
+      const project = await projectEntityRepo.findOneOrFail(id, {
+        select: ["id", "name"]
+      });
+      res.send(project);
+    } catch (error) {
+      res.status(500).send({ error: "Project not found" });
+    }
+  };
+
+  static createProject = async (req: Request, res: Response) => {
+    let { name } = req.body;
+    let project = new Project();
+    project.name = name;
+    project.user = res.locals.userId;
+
+    const errors = await validate(project);
+    if (errors.length > 0) {
+      res.status(400).send(errors);
```

Comment:
It could be better if we use 422 status code. Because request body has valid json syntax and has some validation errors.